Source/Scene/ModelExperimental/ModelExperimental.js

```diff
@@ -534,7 +533,10 @@ Object.defineProperties(ModelExperimental.prototype, {
       }
       //>>includeEnd('debug');
 
-      return this._sceneGraph.boundingSphere;
+      return BoundingSphere.transform(
```

Comment:
Hm, I don't think this is the right place to apply the `transform()`, as this is allocating every time you call the `model.boundingSphere` getter. 

It would probably be better to have `model._boundingSphere` that represents the final bounding sphere, and whenever the model matrix is updated, that bounding sphere should be updated in place (using the `result` parameter).

Reply:
Yeah that makes sense, I think I was trying to incorporate Sean's observation that `model._boundingSphere` wasn't being used and could be deleted. But it makes more sense to actually use that to hold the final bounding sphere, and whenever model matrix is updated, to update that in turn.